sql/expression/function/time_test.go

```diff
@@ -354,6 +354,27 @@ func TestNow(t *testing.T) {
 	require.Equal(t, date, result)
 }
 
+func TestUTCTimestamp(t *testing.T) {
```

Comment:
Should have a test case showing that incorrect values are also handled properly.

Reply:
Don't forget about this